lib/CMakeLists.txt

```diff
@@ -37,8 +47,8 @@ target_include_directories(ziti_tunneler
         PRIVATE ../include/priv
         )
 
-add_library(lwipwin32arch STATIC
-        ${LWIP_CONTRIB_DIR}/ports/win32/sys_arch.c)
+#add_library(lwipwin32arch STATIC
```

Comment:
this needs to be wrapped in `if (WIN32)`

Reply:
Ahh, I meant to un-comment this back in, as I made it conditional in another branch.